backend/src/test/java/com/woowacourse/thankoo/admin/coupon/presentation/AdminCouponControllerTest.java

```diff
@@ -0,0 +1,134 @@
+import static com.woowacourse.thankoo.common.fixtures.MemberFixture.LALA_NAME;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.doNothing;
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.document;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.get;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.put;
+import static org.springframework.restdocs.operation.preprocess.Preprocessors.prettyPrint;
+import static org.springframework.restdocs.payload.JsonFieldType.ARRAY;
+import static org.springframework.restdocs.payload.JsonFieldType.NUMBER;
+import static org.springframework.restdocs.payload.JsonFieldType.STRING;
+import static org.springframework.restdocs.payload.PayloadDocumentation.fieldWithPath;
+import static org.springframework.restdocs.payload.PayloadDocumentation.requestFields;
+import static org.springframework.restdocs.payload.PayloadDocumentation.responseFields;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.woowacourse.thankoo.admin.common.AdminControllerTest;
+import com.woowacourse.thankoo.admin.coupon.application.dto.AdminCouponExpireRequest;
+import com.woowacourse.thankoo.admin.coupon.presentation.dto.AdminCouponResponse;
+import com.woowacourse.thankoo.admin.member.presentation.dto.AdminMemberResponse;
+import java.time.LocalDateTime;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+import org.springframework.restdocs.operation.preprocess.Preprocessors;
+import org.springframework.test.web.servlet.ResultActions;
+
+@DisplayName("AdminCouponController 는 ")
+class AdminCouponControllerTest extends AdminControllerTest {
+
+    @DisplayName("기간에 해당하는 모든 쿠폰을 조회한다.")
+    @Test
+    void getCoupons() throws Exception {
+        AdminMemberResponse lala = new AdminMemberResponse(1L, LALA_NAME, LALA_EMAIL);
+        AdminMemberResponse hoho = new AdminMemberResponse(2L, HOHO_NAME, HOHO_EMAIL);
+        given(adminCouponQueryService.getCoupons(any()))
+                .willReturn(List.of(
+                        new AdminCouponResponse(1L, TYPE, "NOT_USED", lala, hoho,
+                                LocalDateTime.now(), LocalDateTime.now()),
+                        new AdminCouponResponse(2L, TYPE, "RESERVING", hoho, lala,
+                                LocalDateTime.now(), LocalDateTime.now()))
+                );
+
+        ResultActions resultActions = mockMvc.perform(get("/admin/coupons?"
+                        + "status=all"
+                        + "&startDate=2022-01-01"
+                        + "&endDate=2022-12-01"))
+                .andDo(print())
+                .andExpect(status().isOk());
```

Comment:
다음처럼 객체로 사용할 수 있어요!!

.param("status", "all")
.param("startDate", "2022-01-01")

위 같이도 될거에요
```suggestion
        MultiValueMap<String, String> requestParam = new LinkedMultiValueMap<>();
        requestParam.set("status", "all");
        requestParam.set("startDate", "2022-01-01");
        requestParam.set("endDate", "2022-12-01");

        mockMvc.perform(get("/admin/coupons")
                        .params(requestParam))
                .andDo(print())
                .andExpect(status().isOk());
```

Reply:
👍🏻